ocs_ci/ocs/ui/views.py

```diff
@@ -228,6 +228,11 @@
     "check_first_row_checkbox": ('input[name="checkrow0"]', By.CSS_SELECTOR),
     "remove_search_filter": ('button[aria-label="close"]', By.CSS_SELECTOR),
     "delete_resource_kebab_button": ('//*[contains(text(), "Delete")]', By.XPATH),
+    "ocs-storagecluster-storagesystem": (
+        '//a[text()="ocs-storagecluster-storagesystem"]',
```

Comment:
please check this locator from views validation_4_9["ocs-storagecluster-storgesystem"] it should be the same

Reply:
Made the changes.